assets/global_manifest_template.toml

```diff
@@ -0,0 +1,18 @@
+[options.allow]
+# Whether non-FOSS packages should appear in search/install results.
+unfree = true
+
+# Whether "broken" packages should appear in search/install results.
+# These are packages with known build/runtime issues.
+broken = false
+
+# Limit search/install results to only packages which
+# certain SPDX Identifiers by listing them below.
+# Leaving `licenses' undefined allows any license to be used.
+# See all valid license ids here: https://spdx.org/licenses/
+## licenses = ["LGPL2-or-later"]
+
+[options.semver]
+# Whether unstable/pre-release versions of software should be
+# preferred over the latest stable release in search/install results.
+prefer-pre-releases = false
```

Comment:
suggestion nonblocking: the contents of this file appear in every manifest created by `flox init`, right? Most of these options seem niche enough that I think we should make them defaults users don't have to see.

Reply:
I think these exact values are already set as defaults in `pkgdb`: https://github.com/flox/pkgdb/blob/9bbf075a043d153f06baa0c84aa7fd52999987b0/include/flox/pkgdb/pkg-query.hh#L172-L173

I just did what the ticket said and created a global manifest file with the contents that were in the ticket (modulo fixing a typo), then prepend the project manifest piece to the contents of the global manifest. Although re-reading the ticket I can't actually tell whether the intention was to _logically_ extend the global manifest (i.e. don't override any of its behavior in the new `manifest.toml`), or _physically_ extend the global manifest (i.e. append the global manifest contents to the new `manifest.toml`).

Reply:
Yeah, nonblocking, and you did what the ticket said. As it's user facing, I think it's worth considering whether that's really the behavior we want, but we don't need to figure that out now.

Reply:
I've gone ahead and taken out the part that extends the manifest with the actual contents of the global manifest. I don't think that was actually the intent of the ticket.